INN/umbrella-workdayminn
# Add top term to primary and secondary featured posts on archive pages

## Changes

This pull request makes the following changes:

- Created `archive-category-primary-feature.php` and `archive-category-primary-feature.php` partial templates in Workday child theme
- Added `archive.less` for archive specific styles
- Added top term to primary and secondary featured posts on archive pages

![Screen Shot 2019-08-13 at 2 57 39 PM](https://user-images.githubusercontent.com/18353636/62969267-e3919580-bdda-11e9-8931-1fd770ca61fe.png)

## Why



For #65 

## Testing/Questions

Features that this PR affects:

- Archive page featured and secondary posts


Questions that need to be answered before merging:

- [ ] @kaylima @MirandaEcho Do we want to style these in a different way?

Steps to test this PR:

1. View a category archive page
2. Verify top tags are displaying for the featured and secondary featured posts
